atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -149,6 +156,55 @@ def get_buildroot(self, worker_metadatas):
 
         return buildroots
 
+    def get_logs(self):
+        """
+        Build list of log files
+
+        :return: list, of log files
+        """
+
+        logs = None
+        output = []
+
+        # Collect logs from server
+        try:
+            logs = self.osbs.get_orchestrator_build_logs(self.build_id)
+        except OsbsException as ex:
+            self.log.error("unable to get build logs: %r", ex)
+            return output
+        except TypeError:
+            # Older osbs-client has no get_orchestrator_build_logs
+            self.log.error("Your osbs client doesn't support this functionality")
```

Comment:
Maybe it's worth describing what the functionality is?

Reply:
Will make that message less ambiguous and more useful.